components/AppToolbar.vue

```diff
@@ -1,9 +1,23 @@
 <script setup type="ts">
 const route = useRoute();
 const hasHistory = () => window.history.length > 1; // using a computed() doesn't work on inital load in the Electron app. No idea why. See PR #433
+
+onMounted(() => {
```

Comment:
looks unfinished

Reply:
This is for the lyrics editor, to make sure the "Verse title" button sticks to the bottom of the apps toolbar

Reply:
but why do you call onMounted without any code in it?

Reply:
Ah yes, that is wrong 😂 Will remove